src/clj_sqs_extended/serdes.clj

```diff
@@ -39,13 +39,17 @@
     (= format :transit) (transit-write out)
     (= format :json) (json/generate-string out)
     (= format :raw) out
-    :else out))
+    :else (throw (ex-info "Format not supported"
```

Comment:
error message should guide user on how to fix the error. i.e. `(ex-info (format "Only %s formats are supported. We received %s." supported-formats format))`